compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeBridge.desktop.kt

```diff
@@ -383,16 +383,27 @@ internal class ComposeBridge(
     }
 
     fun updateSceneSize() {
+        // Zero size will literally limit scene's content size to zero,
+        // so it case of late initialization skip this to avoid extra layout run.
```

Comment:
Zero size can be not only on initialization, but also on usual resize. Not applying zero size in this case can lead to state incosistency, and then to issues.

So, I wouldn't filter the zero size. If we need to avoid zaero size at the initialization state, we should do it elsewhere.

Reply:
Let's discuss it separately, here it's just moving lines around.